pandas/core/frame.py

```diff
@@ -8863,20 +8863,20 @@ def update(
         >>> df = pd.DataFrame({'Animal': ['Falcon', 'Falcon',
         ...                               'Parrot', 'Parrot'],
         ...                    'Max Speed': [380., 370., 24., 26.]})
-        >>> df.groupby("Animal", group_keys=True).apply(lambda x: x)
-                  Animal  Max Speed
+        >>> df.groupby("Animal", group_keys=True)[['Max Speed']].apply(lambda x: x)
+                  Max Speed
```

Comment:
Does `df.groupby("Animal", group_keys=True).apply(lambda x: x, include_groups=False)`return a Series or a DataFrame? (should return a DataFrame, like groupby.transform IMO.

Reply:
A DataFrame